packages/manager/src/features/CloudPulse/Alerts/AlertsResources/AlertsResources.tsx

```diff
@@ -336,7 +336,8 @@ export const AlertResources = React.memo((props: AlertResourcesProp) => {
     );
   }
 
-  const filtersToRender = serviceToFiltersMap[serviceType ?? ''];
+  const filtersToRender =
+    serviceToFiltersMap[serviceType ?? ''] ?? serviceToFiltersMap[''];
```

Comment:
```suggestion
    serviceType ? serviceToFiltersMap[serviceType] ?? [] : [];
```

for undefined service type, for the statements will give undefined value which will break wherever that variable is used

Reply:
That case is already handled within the 
```ts
serviceToFilterMap[serviceType ?? ''] 
```

The reason why this was breaking was that for unsupported services, the serviceType won't be undefined it will be a string type, which the serviceToFilterMap won't have any fallback. So added the `serviceToFiltersMap['']` as fallback to that.